ui/src/app/pages/workspace/research-purpose.tsx

```diff
@@ -41,51 +56,83 @@ export const ResearchPurpose = withCurrentWorkspace()(
       return <FadeBox>
-        <div style={styles.mainHeader}>Research Purpose
+        <div style={styles.mainHeader}>Primary purpose of project
           <Clickable disabled={!workspacePermissions.canWrite}
+                     style={{display: 'flex', alignItems: 'center'}}
                      data-test-id='edit-workspace'
                      onClick={() => navigate(
                        ['workspaces',  workspace.namespace, workspace.id, 'edit'])}>
             <EditComponentReact enableHoverEffect={true}
                                 disabled={!workspacePermissions.canWrite}
-                                style={{marginTop: '0.1rem'}}/>
+                                style={{marginTop: '0.1rem', height: 22,
+                                  width: 22,
+                                  fill: colors.light,
+                                  backgroundColor: colors.accent,
+                                  cursor: 'pointer',
+                                  padding: '5px',
+                                  borderRadius: '23px'}}/>
           </Clickable>
         </div>
-        <div style={styles.sectionText}>{ResearchPurposeDescription}</div>
-        <div style={styles.sectionHeader}>Primary purpose of the project</div>
-        <TwoColPaddedTable header={false} style={{marginTop: '0.3rem'}}
-                           contentLeft={selectedResearchPurposeItems.slice(0, rpItemsHalfLen)}
-                           contentRight={selectedResearchPurposeItems.slice(rpItemsHalfLen)}/>
-        <div style={styles.sectionHeader}>
-          Reason for choosing <i>All of Us</i> data for your investigation</div>
-        <div style={styles.sectionText}>{workspace.researchPurpose.reasonForAllOfUs}</div>
-        <div style={styles.sectionHeader}>Area of intended study</div>
-        <div style={styles.sectionText}>{workspace.researchPurpose.intendedStudy}</div>
-        <div style={styles.sectionHeader}>Anticipated findings from this study</div>
-        <div style={styles.sectionText}>{workspace.researchPurpose.anticipatedFindings}</div>
-        {workspace.researchPurpose.population && <div style={{marginBottom: '1rem'}}>
-          <div style={styles.sectionHeader}>Population area(s) of focus</div>
-          <TwoColPaddedTable header={false} style={{marginTop: '0.3rem'}}
-                             contentLeft={this.getSelectedPopulationsSlice(true)}
-                             contentRight={this.getSelectedPopulationsSlice(false)}/>
+        <div style={styles.sectionContentContainer}>
+          {selectedResearchPurposeItems.map((selectedResearchPurposeItem, i) => <div key={i}>
+            <div style={{marginTop: '1rem'}}>{selectedResearchPurposeItem}</div>
+          </div>)}
+        </div>
+        <div style={styles.sectionHeader}>Summary of research purpose</div>
+        <div style={styles.sectionContentContainer}>
+          <div style={styles.sectionSubHeader}>{researchPurposeQuestions[2].header}</div>
```

Comment:
nit: It will be good to have comments to identify what section the divs are displaying(intended study or anticipated finding)

Reply:
However it did take me 2-3 times to read the whole thing to understand where the section starts and end, can we have just some space distinguishing each section (header and text field) to make it readable

Reply:
Is this good to ignore? Just checking? (If it is, do you mind resolving the comment?)

Reply:
Ignore comment just add space so its easier to read

Reply:
Done, have separated!